src/mail/lib/mail.js

```diff
@@ -15,6 +15,14 @@ nunjucks.configure(templatesDir);
 
 const images = fs.readdirSync(imagesDir);
 
+/**
+ * Generate a mail
+ *
+ * @param {String} templateName - Name of template
+ * @param {*} locals - Local of mail
```

Comment:
`nunjucks.render` needs an object, not **any**thing
```suggestion
 * @param {Object} locals - Local of mail
```